pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -918,6 +918,24 @@ self: super: {
     '';
   }) super.structured-haskell-mode;
 
+  csound-expression-dynamic =
+    assert super.csound-expression-dynamic.version == "0.3.9";
+    overrideCabal (drv: {
+      src = (pkgs.fetchFromGitHub {
+        owner = "spell-music";
+        repo = "csound-expression";
+        rev = "345df2c91c9831dd895f58951990165598504814";
+        hash = "sha256-6qPiKsZwZpqB2kmckKDKyQPTcWPIaVwi+EYs74tRod0=";
+      }) + "/csound-expression-dynamic";
+      editedCabalFile = null;
+      broken = false;
```

Comment:
Instead of setting `broken = false` here, please remove `csound-expression-dynamic` from `pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml` and run `maintainers/scripts/haskell/regenerate-hackage-packages.sh`

Reply:
@zeuner How did you run the maintainer script? It has a nix-shell prompt which provides hackage2nix. That error is quite unusual to me.

Reply:
@maralorn My bad. I ran it as `bash maintainers/scripts/haskell/regenerate-hackage-packages.sh`, thereby disrespecting the shebang. This led to some packages being built, but apparently the required environment wasn't completely built.

Should be fixed now.

Reply:
Yeah, I suspected something like that.